packages/lib/__tests__/fieldInteraction.tsx

```diff
@@ -141,7 +141,7 @@ describe('Field interactions', async () => {
     });
   });
 
-  it('should be able to use formAPI', () => {
+  xit('should be able to use formAPI', () => {
```

Comment:
Why ? :))

Reply:
This test checks if `formApi` is usable from the adapterProps.
But this PR removes passing `formApi` to props